client/python/armada_client/client.py

```diff
@@ -224,6 +224,26 @@ def update_queue(
         )
         self.submit_stub.UpdateQueue(request)
 
+    def send_create_queue_request(self, queue: submit_pb2.Queue) -> empty_pb2.Empty:
```

Comment:
This RPC call isn't really consistent with the names of the other calls.  I'd probably just rename this to create_queue

Reply:
But we already have a method named `create_queue`. Do we want to replace it? that is a massive breaking change IMO

Reply:
What is the end goal?  We want to replace create_queue with this call eventually?

We don't have any users other than airflow client so I'm not that concerned with a breaking change.  We have releases to pypi so someone could use an older release if that was a big deal.

I do like that you are thinking about it but queue creation isn't really a hotspot for our users.

Reply:
Well I think the end goal is to remove `create_queue` as well and soley use `create_queues`. But if this is acceptable, I will work on the replacement process now :))